acra/security-controls/encryption/_index.md

```diff
@@ -2,3 +2,129 @@
 title: Encryption
 bookCollapseSection: true
 ---
+
+# Encryption
+
+Data encryption may be the main feature provided by Acra,
+even though there are plenty of other useful tools, such as SQL firewall.
+This page describes how to configure it and what you will get as a result.
+
+## Behavior
+
+AcraServer works as a proxy between database client and the database itself.
+When a column is configured to be encrypted, AcraServer will transparently encrypt/decrypt its value.
```

Comment:
actually column may not be configured to be encrypted but transparently decrypted by Acra that recognizes AcraStructs. App can encrypt data with AcraWriter and save it directly to database. And query data through Acra that will be decrypted transparently without additional configuration.